src/main/groovy/de/gesellix/gradle/docker/models/DockerContainer.groovy

```diff
@@ -0,0 +1,393 @@
+package de.gesellix.gradle.docker.models
```

Comment:
Would you expect that we'll get more models? If so, what's a model? I'd propose to rename the package to `containers`. The `DockerContainer` is not only a model (which would be a simple value object for me) but the main working horse.